test/integration/centos/inspec/centos_spec.rb

```diff
@@ -0,0 +1,79 @@
+# Test for the main configuration file ('/etc/yum.conf'cookstyle)
+describe ini('/etc/yum.conf') do
+  its('main.installonlypkgs') { should eq 'kernel kernel-osuosl' }
+  its('main.installonly_limit') { should eq '2' }
+end
+
+# There will be different cases for the Centos 7 and Centos 8 repositories
+case os.release.to_i
+
+# Begin Centos 7 Case
+when 7
+
+  # Test the base repository
+  describe yum.repo('base') do
+    it { should exist }
+    it { should be_enabled }
+    its('baseurl') { should eq 'https://centos.osuosl.org/7/os/x86_64/' }
```

Comment:
I'd really like to be able to use this on the other architectures we support.
```suggestion
    its('baseurl') { should eq "https://centos.osuosl.org/7/os/#{arch}/" }
```
You should probably make a variable like the following:
```ruby
arch = os.arch
```

Now for testing on POWER9, you might want to use the following for the logic:
```ruby
power9 = File.readlines('/proc/cpuinfo').grep(/POWER9/).any?
```

Reply:
will do, i didnt realize we ran inspec tests outside of our x86 environment

Reply:
@ramereth can i test other architectures easily to make sure this works? specefically power9

Reply:
It's not exactly easy as it requires access to each cluster. It's best if you do what you think is right and I can test it on my end.

Reply:
okay thats done then